account_banking_ach_base/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Thinkwell Designs <dave@thinkwelldesigns.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Localizations for North American Banking & Financials",
+    "summary": "Add fields required for North American Banking & Financials",
+    "version": "16.0.1.0.1",
+    "license": "AGPL-3",
+    "author": "Thinkwell Designs, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-usa",
+    "category": "Banking addons",
+    "depends": [
+        "account_payment_order",
+        "account_banking_mandate",
+        "l10n_us_partner_legal_number",
+    ],
+    "data": [
+        "views/account_banking_mandate.xml",
```

Comment:
According to [Odoo's guidelines](https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#file-naming) which are the base for ours this file's name should be suffixed with "_views". The renaming should be done with "git move" command.